daemon/daemon/daemon.go

```diff
@@ -421,13 +423,16 @@ func NewDaemon(c *Config) (*Daemon, error) {
 		endpointsDockerEP:         make(map[string]*endpoint.Endpoint),
 		endpointsLearning:         make(map[uint16]labels.LearningLabel),
 		endpointsLearningRegister: make(chan labels.LearningLabel, 1),
-		loadBalancer:              lb,
-		consumableCache:           policy.NewConsumableCache(),
-		policy:                    policy.Tree{Root: policy.NewNode(common.GlobalLabelPrefix, nil)},
-		ignoredContainers:         make(map[string]int),
+		events:            make(chan events.Event, 10),
```

Comment:
I'm not worried about listeners yet, the `listenForCiliumEvents()` itself needs to be scheduled to run to drain the 10 events before the main daemon continues running.